package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Serverless Framework, Apollo Server, and Contentful CMS",
   "main": "src/handlers/graphql.ts",
   "scripts": {
-    "dev": "sls offline start",
+  	"dev": "sls offline start",
     "test": "npm run codegen && jest",
     "lint": "eslint . --ext .ts",
     "deploy": "sls deploy",
     "format": "prettier --write \"**/*.{ts,js,json,md}\"",
     "ts:check": "tsc --noEmit",
-    "codegen": "graphql-codegen --config codegen.ts",
-    "db:seed": "npm run codegen && ts-node ./scripts/seedData.ts"
+    "codegen": "ts-node -r tsconfig-paths/register node_modules/.bin/graphql-codegen --config codegen.ts",
```

Comment:
For the codegen, this can be added in the config `codegen.ts`
Just add an extra:
```
{
...
require: ['tsconfig-paths/register'],
}
```

Reply:
oh nice!